packages/shared-components/src/components/onto-dropdown/onto-dropdown.tsx

```diff
@@ -65,6 +66,14 @@ export class OntoDropdown {
    */
   @Prop() tooltipTheme: string;
 
+  /**
+   * Specifies the items tooltip placement. Accepts a string of the placement or a function that returns the placement.
+   * The function takes the isOpen parameter as a boolean and returns the placement as a string.
+   * If not provided, the tooltip will be placed to the left.
+   *
+   */
+  @Prop() tooltipPlacement: string | ((isOpen: boolean) => void) = 'left';
```

Comment:
Can we have some constant with strings union maybe for the placement variations we are going to support?

Reply:
Added `OntoTooltipPlacement`